### ISSUE TITLE
[Issue]: External subtitles are not displayed properly (10.8.1)

### ISSUE THREAD

**Issue by CodeName393** [jellyfin/jellyfin#8022]
### Please describe your bug

After updating to 10.8.1, when I check if subtitles are displayed properly on Jellyfin mobile, subtitles are not displayed properly.
Selecting an external SUBRIP subtitle will stream the video itself.

Add some contents.
The exoplayer also does not have a subtitle, but there is no error in the log.
An error is displayed when playing with an external player.

### Jellyfin Version

Other

### if other:

10.8.1

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
[14:50:28] [INF] [166] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile=Android External Player Stub, Path=video.mkv, AudioStreamIndex=2, SubtitleStreamIndex=0 ) => ( PlayMethod=Transcode, TranscodeReason=SubtitleCodecNotSupported ) media:/videos/2c75a6b1-a2fc-dd26-4986-bd70c09635f7/stream?MediaSourceId=2c75a6b1a2fcdd264986bd70c09635f7&AudioStreamIndex=2&SubtitleStreamIndex=0&api_key=<token>&SubtitleMethod=Encode&RequireAvc=false&Tag=507a1ef156a45c491c4d119ceaf11e02&TranscodeReasons=SubtitleCodecNotSupported
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mmtech2012** [comment]
I have this bug too since 10.8.0

**Comment by jtowe1** [comment]
Having the same issue in `10.8.1`

Playing in chrome, the subtitle icon just doesn't appear.
Playing in roku app Version 1.4 build 12, I can see and select the subtitle, but then it goes back to the movie page with "Error During Playback: An error was encountered while playing this item"

This is the log from when I select a subtitle while playing in roku
```
today at 7:20:25 PM[16:20:25] [INF] [87] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for tehjrow. 
EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True

today at 7:20:25 PM[16:20:25] [INF] [87] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( 
Profile=Anonymous Profile, Path=/data/movies/Raiders of the Lost Ark (1981)/Raiders of the Lost Ark (1981) Bluray-
1080p.mkv, AudioStreamIndex=1, SubtitleStreamIndex=2 ) => ( PlayMethod=Transcode, 
TranscodeReason=SubtitleCodecNotSupported ) media:/videos/e7117163-9c92-ce2e-ed0f-c5cea1c428d7/master.m3u8?
MediaSourceId=e71171639c92ce2eed0fc5cea1c428d7&VideoCodec=h264,mpeg2video,h265,hevc,hevc&AudioCodec=ac3
&AudioStreamIndex=1&SubtitleStreamIndex=2&VideoBitrate=139552000&AudioBitrate=448000&AudioSampleRate=48000
&MaxFramerate=23.976025&api_key=
<token>&SubtitleMethod=Encode&TranscodingMaxAudioChannels=6&RequireAvc=false&Tag=105a09a3197bd075433882c
963eba95c&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&hevc-level=120&hevc-
videobitdepth=8&hevc-profile=main&hevc-audiochannels=6&h264-profile=high,main,baseline,constrainedbaseline&h264-
level=51&aac-audiochannels=6&TranscodeReasons=SubtitleCodecNotSupportedtoday at 7:20:25 PM[16:20:25] [INF] [24] 
Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app Jellyfin Roku 1.4.12 playing 
Raiders of the Lost Ark. Stopped at 0 ms
```

**Comment by jtowe1** [comment]
This problem went away for me in 10.8.3

**Comment by prahal** [comment]
@mmtech2012 is the external subtitle detection issue also fixed for you in Jellyfin 10.8.3?

@CodeName393 do you also still experience this bug with Jellyfin 10.8.3? Could you close the bug if it is already fixed?

**Comment by mmtech2012** [comment]
@prahal its fixed thank you
